infrastructure/static-site/maintenance/nginx.conf

```diff
@@ -0,0 +1,23 @@
+worker_processes 1;
+
+events {
+    worker_connections 1024;
+}
+
+pid /usr/share/nginx/tmp/nginx.pid;
+
+http {
+    client_body_temp_path /usr/share/nginx/tmp/client_temp;
+    proxy_temp_path /usr/share/nginx/tmp/proxy_temp;
+    fastcgi_temp_path /usr/share/nginx/tmp/fastcgi_temp;

```

Comment:
why do we need to have these? just a static site, wouldn't think we need any cgi stuff.

Reply:
Just standard copy n' paste path fixens from Chat to resolve the permission issues.